tests/test_dat.py

```diff
@@ -14,3 +14,20 @@ def test_dat_load1(data_folder):
     data = dat.load(count=90)
     chunk = dat.load(count=9, offset=21)
     assert np.allclose(chunk, data[:, :, 21:30])
+
+
+def test_dat_chunks(data_folder):
+    dat = Dat(data_folder)
+    chunk_size = 9
+    for chunk in dat.chunks(chunk_size=chunk_size):
+        assert chunk.shape == (dat.rows * 2, dat.cols, chunk_size)
+
+
+def test_dat_chunks1(data_folder):
```

Comment:
I feel it's better to combine the two tests into one (the first is passing as a special case and will stop passing if we ever tweak the sample data), and use a random chunk size. Something like this:

```
    chunk_size = np.random.randint(0, 20)

    for chunk in dat.chunks(chunk_size=chunk_size):
        last_iteration = False
        try:
            assert chunk.shape == (dat.rows * 2, dat.cols, chunk_size)
        except AssertionError:
            last_iteration = True

    if last_iteration:
        remainder = dat.n_frames % chunk_size
        assert chunk.shape == (dat.rows * 2, dat.cols, remainder)
```

Reply:
This will also prevent a false negative that we might get owing to the `or` condition (a `.chunks()` that simply handed out `remainder` size chunks would pass that test).

Reply:
Actually when I think about it, the following statement is true even of the new code. How about:
```
    chunks = dat.chunks(chunk_size=chunk_size)
    for chunk in chunks:
        try:
            assert chunk.shape == (dat.rows * 2, dat.cols, chunk_size)
        except AssertionError:
            try:
                next(chunks)
            except StopIteration:
                assert chunk.shape == (dat.rows * 2, dat.cols, dat.n_frames % chunk_size)
            else:
                raise RuntimeError("Expected last iteration")
```

Reply:
Updated test case accordingly. Added another one that tests the same thing but on a different manner.